src/xunit.performance.analysis/Program.cs

```diff
@@ -41,7 +45,7 @@ private static int Main(string[] args)
 
             for (int i = 0; i < args.Length; i++)
             {
-                if (args[i].StartsWith("-") || args[i].StartsWith("/"))
```

Comment:
are you removing this because it was interfering with some other parameters, or because other tools do not support the '/' notation?